docker/docker-compose.yml

```diff
@@ -28,6 +28,22 @@ services:
       - AWS_SECRET_ACCESS_KEY
       - AWS_REGION
 
+  mock-mhs-adaptor:
+    build:
+      context: ../
+      dockerfile: docker/mock-mhs-adaptor/Dockerfile
+    depends_on: 
+      - gp2gp
+    ports:
+      - "8081:8081"
+    environment:
+      - GP2GP_SERVER_PORT=8081
+      - GP2GP_LOGGING_LEVEL
+      - GP2GP_AMQP_BROKERS=amqp://activemq:5672
+      - GP2GP_AMQP_USERNAME
+      - GP2GP_AMQP_PASSWORD
+      - GP2GP_AMQP_MAX_REDELIVERIES
```

Comment:
I think we'll need to be able to specify the inbound queue name with an environment variable also